wwwapp/models.py

```diff
@@ -281,3 +289,23 @@ def result_in_percent(self):
 
     class Meta:
         unique_together = [('workshop', 'participant')]
+
+
+class ResourceYearPermission(models.Model):
```

Comment:
Documentation

Reply:
What about changing the name to CampResourcePermission - we will probably want to introduce a Camp model soon in order to avoid hardcoding the dates and the year in the settings.

Reply:
I don't actually like either of these names - nothing prevents you from having multiple resources per year/camp and we could possibly introduce public resources at some point (e.g. photos). I would just call it `ResourcePermission` or even simply `Resource` (since it's not only about the permissions but also about displaying it in the menu)

Reply:
I would separate Resource and it's permissions in the future. Introducing public resources and complicated permissions is not really needed right now. This is really only designed for "INTERNETY". Even the menu dropdown title is `INTERNETY`. Public resources are and probably will in the future be hosted separately.

Reply:
Please don't use word "camp" as a name for the workshops :)